tests/foreman/cli/test_discoveryrule.py

```diff
@@ -240,75 +283,37 @@ def test_positive_create_disabled_rule(self, discoveryrule_factory):
         rule = discoveryrule_factory(options={'enabled': 'false'})
         assert rule.enabled == 'false'
 
-    @pytest.mark.tier3
+    @pytest.mark.tier1
+    @pytest.mark.parametrize('host_name', **parametrized(invalid_hostnames_list()))
```

Comment:
host_name doens't seem to be used in test, is this intended? and this will unnecessary comination of parametrized tests, please check?

Reply:
it is relevant and being used in the test for forming combinations of the invalid host_name